domains/joshua.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Joshua's personal developer website",
+    "repo": "https://github.com/JoshuaCarroll",
+    "owner": {
+        "username": "joshuacarroll",
+        "email": "joshcarroll@hotmail.com"
+    },
+    "record": {
+        "CNAME": "aa5jc.com",
+        "URL": "http://aa5jc.com"
```

You should have either a cname to your domain or redirect to your domain.

**If you just want to redirect to your domain**
```suggestion
        "URL": "http://aa5jc.com"
```

**If you want to add a cname with forced https**
```suggestion
        "CNAME": "aa5jc.com",
        "URL": "https://joshua.is-a.dev"
```